### DISCUSSION TITLE
Add COALESCE statement support to Linq

### DISCUSSION THREAD

**Discussion by KenshinRus** [JasperFx/marten#2020] [Q&A]
Looks like there is nothing available in Marten linq extensions to support coalescing operator ?? when select fields, aka
```
COALESCE(d.data -> 'LastApprovedRequest' ->> 'Status', d.data ->> 'Status')
```

**Comment by jeremydmiller** [comment]
Correct. That's never come up before. Think this is an "I take pull requests" moment.
